ribs/emitters/_gaussian_emitter.py

```diff
@@ -43,17 +47,18 @@ class GaussianEmitter(EmitterBase):
     def __init__(self,
                  archive,
                  x0,
-                 sigma0,
-                 sigma=None,
+                 sigma,
+                 sigma0=None,
                  bounds=None,
                  batch_size=64,
                  seed=None):
         self._rng = np.random.default_rng(seed)
         self._batch_size = batch_size
         self._x0 = np.array(x0, dtype=archive.dtype)
-        self._sigma0 = archive.dtype(sigma0) if isinstance(
-            sigma0, (float, np.floating)) else np.array(sigma0)
-        self._sigma = sigma0 if sigma is None else sigma
+        self._sigma = archive.dtype(sigma) if isinstance(
+            sigma,
+            (float, np.floating)) else np.array(sigma, dtype=archive.dtype)
+        self._sigma0 = sigma if sigma0 is None else sigma0
```

Comment:
Note that we do some conversions above for `self._sigma` so that it has the correct `dtype`. Thus, over here, if sigma0 is None, then set it to `self._sigma` (not `sigma`). Otherwise, set it to the correct dtype just like `self._sigma` does.